releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,15 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 3.0.0 - February 19, 2020
+* Added the following method to allow users to override `ConfigurationClientOptions`. These can be used to update the default retry options or configure a proxy for connecting to Azure App Configuration.
+
+   ````csharp
+   public AzureAppConfigurationOptions ConfigureClientOptions(Action<ConfigurationClientOptions> options)
+   ````
+* Added `IConfigurationRefresher.TryRefreshAsync` method to make it easier to trigger configuration refresh without crashing the application due to transient errors. Also, renamed the `Refresh` method to `RefreshAsync`.
+* Reduced the number of retry attempts made for failed requests on initial configuration load from 12 to 2. This prevents manual refresh from blocking the application for long periods of time.
```

Comment:
Can we shorten it a bit? Something like "Reduced maximum number of retries when query App Configuration to prevent blocking the application for long periods of time during refresh."

Reply:
Add a link to the issue too.
https://github.com/Azure/AppConfiguration/issues/255